config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
   root 'static_pages#home'
-  resources :works, only: [:index, :show]
+  resources :works, :only => [:index, :show, :new, :create]
```

Comment:
Rubocop will want this to use the newer syntax (`only: [:index, :show, :new, :create]`). I'd merge this with master (which now includes Rubocop) and run the checks.